### DISCUSSION TITLE
Any chance to get a fixed volume/output over DLNA or Slimproto to go into a headphone amp?

### DISCUSSION THREAD

**Discussion by milkplus27** [music-assistant/support#2876] [Q&A] [answered]
Pretty much as the title suggests. I'd like to know if it's possible to have a fixed volume output from my RPi into a headphone amp that I can control the volume from. Couldn't find anything in setting or in the documentation.

**Comment by OzGav** [comment]
What about https://music-assistant.io/faq/stream-to/#my-local-ha-device

**Comment by milkplus27** [comment]
Thanks. But this wasn't what I'm actually after. I should have been a bit
clearer perhaps - I'm not streaming from my HA machine but actually a
separate RPi that's running PiCore Player. That's then connected to the
headphone amp via USB. At the moment I'm just setting the volume in MA to
about 75% but was wondering if it's possible to have it as a fixed output.

**Reply by OzGav** [reply]
No that isn't possible. Out of interest what problem does it cause just having the volume at 75%?

**Comment by milkplus27** [comment]
Oh, there's no issue really :) just that the volume might get bumped
accidentally and clip in the headphone amp but that's not really
prohibitive to anything haha

**Reply by OzGav** [reply, marked as answer]
If you are using HA you could set an automation to set it back to 75 if it changes

**Comment by milkplus27** [comment]
I am using HA. And that is an excellent idea! Cheers
